### PR TITLE
fix(model-server): make initial branch created through V2 API visible in V1 API

### PR DESCRIPTION
**Pull request by odzhychko**
Writing to a branch through V2 API already makes a branch visible through the V1 API. Making the branch visible after initial creation makes the backwards compatibility more consistent.

I found this issue while reviewing and testing #714

### CODE REVIEW

**Comment by languitar** [comment]
This should probably be a draft until a rebase after merging #714?

File: model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerBackwardsCompatibilityTest.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            keyValueLikeModelServer.init(this)
+        }
+
+        coroutineScope {
+            block(this)
+        }
+    }
+
+    @Test
+    fun `master branch of repository initialized by V2 API is visible through V2 API`() {
```
Comment (languitar) [inline model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerBackwardsCompatibilityTest.kt:61]: One of them should be V1?

**Review by languitar — CHANGES_REQUESTED** [review]

File: model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerBackwardsCompatibilityTest.kt (hunk 1 above)
Comment (odzhychko) [reply model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerBackwardsCompatibilityTest.kt]: Yes. Done.

**Review by languitar — APPROVED** [review]

**Pull request merged**